src/coreclr/jit/compiler.cpp

```diff
@@ -10622,7 +10622,7 @@ const char* Compiler::convertUtf16ToUtf8ForPrinting(const WCHAR* utf16String)
 {
     const char* utf8Str = "<utf8 conversion failure>";
     int         utf8Len = WideCharToMultiByte(CP_UTF8, 0, utf16String, -1, nullptr, 0, nullptr, nullptr);
-    if (utf8Len == 0)
+    if (utf8Len > 0)
```

Comment:
Convert the containing function to use the minipal APIs directly instead of going through the Win32 PAL?

Reply:
@jkoritzinsky which minipal function are you referring to?

Reply:
Rewrote to PAL